cli/client.go

```diff
@@ -380,6 +387,129 @@ func RobotsPartShellAction(c *cli.Context) error {
 	)
 }
 
+// checkUpdateResponse holds the values used to hold release information.
+type checkUpdateResponse struct {
+	Name       string `json:"name"`
+	TagName    string `json:"tag_name"`
+	TarballURL string `json:"tarball_url"`
+}
+
+func checkLatestRelease() (checkUpdateResponse, error) {
```

Comment:
function name should be `getLatestReleaseVersion` or something like that, and it should only return tag name - doesn't seem like the other fields are used, so don't return it